twisted/plugins/multichain-crawler.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description = Multichain Crawler
+After = network-online.target
+
+[Service]
+Type = simple
+Environment="PYTHONPATH=<path to tribler checkout>"
```

Comment:
Make that point to /opt/tribler so it works by default when deploying via ansible. It's trivial to override via a drop in file.